demonstrations/tutorial_how_to_use_arithmetic_operators.py

```diff
@@ -0,0 +1,343 @@
+
+    # We return the x and y registers to its original value using the adjoint operation
+    # |5x>|3y>|5x + 3y> --->  |x>|y>|5x + 3y>
+    qml.adjoint(qml.Multiplier)(5, wires["x"], work_wires=wires["work_wires"])
+    qml.adjoint(qml.Multiplier)(3, wires["y"], work_wires=wires["work_wires"])
+
+######################################################################
+# In this functions, we showed how to undo the multiplication by using the adjoint operation, which in this case is useful to clean
+# the input registers ``wires["x"]`` and ``wires["y"]`` after the operation.
+#
+# Now we can combine all these functions to load the function :math:`f(x,y)= 4 + 3xy + 5 x+ 3 y` into a quantum state.
+@qml.qnode(dev)
+def circuit(x,y):
+
+    prepare_initial_state(x, y)    #    |x> |y> |0>
+    qml.Adder(4, wires["output"])  #    |x> |y> |4>
+    adding_3xy()                   #    |x> |y> |4 + 3xy>
+    adding_5x_3y()                 #    |x> |y> |4 + 3xy + 5x + 3y>
+
+    return qml.sample(wires=wires["output"])
+
+print(circuit(x=1,y=4), " which represents the number ", state_to_decimal(circuit(x=1,y=4)))
+
+######################################################################
+# Cool, we get the correct result 33!
+#
+# At this point, it's interesting to consider what would happen if we had chosen a smaller number of wires for the output.
+# For instance, if we had selected one less wire for the output, we would have obtained the result :math:`33 \mod 2^5 = 1`.
+
+wires = qml.registers({"x": 4, "y": 4, "output": 5, "work_wires": 4})
+print(circuit(x=1,y=4), " which represents the number ", state_to_decimal(circuit(x=1,y=4)))
+
+######################################################################
+# With one less wire, we get :math:`1`, just like we predicted. Remember, we are working with modular arithmetic!
+#
+# Using OutPoly
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# In the last section, we showed how to use different arithmetic operations to load 
+# a function onto a quantum computer. But what if I told you there's an easier way to do all this using just one
+# PennyLane function that handles the arithmetic for you? I'm talking about :class:`~.pennylane.OutPoly`. 
+# This handy operator lets you load polynomials directly into quantum states, taking care of all the arithmetic in one go. 
+# Let's check out how to load a function like :math:`f(x, y)` using :class:`~.pennylane.OutPoly`.
+#
+# We will start by explicitly defining our function:
+
+def f(x, y):
+   return 4 + 3*x*y + 5*x + 3*y
+
+######################################################################
+# Now, we load it into a quantum circuit.
+
+######################################################################
+
+wires = qml.registers({"x": 4, "y":4, "output":6,"work_wires": 4})
+@qml.qnode(dev)
+def circuit_with_Poly(x,y):
+
+   prepare_initial_state(x, y)
+   qml.OutPoly(f, registers_wires=[wires["x"], wires["y"], wires["output"]])
```

Comment:
Based on new interface 

```suggestion
wires = qml.registers({"x": 4, "y":4, "output":6,"work_wires": 4})
@qml.qnode(dev)
def circuit_with_Poly(x,y):

   prepare_initial_state(x, y)
   qml.OutPoly(
       f, 
       x_wires = wires["x"],
       y_wires = wires["y"],
       output_wires = wires["output"])
```